gapic-common/gapic-common.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.4"
 
-  spec.add_dependency "google-protobuf", "~> 3.2"
+  spec.add_dependency "google-protobuf", "~> 3.12", ">=3.12.2"
```

Comment:
Nit: space between `>=` and `3.12.2`. (I think it will parse okay without it, but better to be safe.)